### PR TITLE
Updated default token symbol to match light wallet (#672)

### LINKED ISSUE #672: `token generation` - update
#### What would you like to be added:
Update token generation to match light wallet - https://github.com/DeFiCh/scan/blob/main/src/components/icons/assets/_Default.tsx

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #672

### CODE REVIEW

File: app/components/icons/assets/_Default.tsx [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
 import * as React from 'react'
-import Svg, { Circle, SvgProps, Text } from 'react-native-svg'
+import randomColor from 'randomcolor'
+import { View, Text } from 'react-native'
+import { tailwind } from '../../../tailwind'
+
+interface _DefaultProps extends React.SVGProps<SVGSVGElement> {
+  testID?: string
+  style?: object
+}
```
Comment (fuxingloh) [inline app/components/icons/assets/_Default.tsx:9]: these default props will never be able to be passed in as on the upstream, only `SVGProps<SVGSVGElement>` is exposed



https://github.com/DeFiCh/wallet/blob/c60d9f5f9a8855a5b1f6b4ecf0ad032f5529c985/app/components/icons/assets/index.ts#L37

Comment (fullstackninja864) [reply app/components/icons/assets/_Default.tsx:9]: Thanks for the comment, I will make changes accordingly

**Comment by thedoublejay** [comment]
@fullstackninja864 app doesn't open on mobile with this build

File: app/components/icons/assets/_Default.tsx [hunk 2]
```diff
@@ -1,21 +1,30 @@
...
+export function _Default (symbol: string): (props: _DefaultProps) => JSX.Element {
+  return (props: _DefaultProps): JSX.Element => {
+    const { style, testID } = props
+    const height = '32px'
+    const width = '32px'
+    const bg = randomColor({ luminosity: 'bright', format: 'rgba', seed: symbol, alpha: 0.2 })
+    const text = randomColor({ luminosity: 'dark', format: 'rgba', seed: symbol, alpha: 100 })
+    const first = symbol?.substring(0, 1)
 
-export function _Default (symbol: string): (props: SvgProps) => JSX.Element {
```
Comment (thedoublejay) [inline app/components/icons/assets/_Default.tsx:5]: I suggest we keep the SVG element here. It's rendered differently in mobile. Also, `px` doesn't work on mobile

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**